task_assign/waypoint[] path_a
int32 id_a

task_assign/waypoint[] path_b
int32 id_b

task_assign/task t_name
task_assign/robot r_name
bool stop
